### DISCUSSION TITLE
o xadrez não aparece de jeito nenhum

### DISCUSSION THREAD

**Discussion by ClaraKataryna654** [Monika-After-Story/MonikaModDev#8084] [QA - tech support + general] [answered]
eu já tentei de tudo já tentei colocar outros arquivos de xadrez, fiquei abrindo e fechando o jogo varias vezes mais mesmo assim não fusionou. Alguém pode me ajudar ?

**Comment by satoshi-kun** [comment, marked as answer]
Chess and other things are unlocked through affection, so you need to spend time with Monika before that.
De futuro quando tiveres dúvidas escreve-as em inglês.
As chances de te responderem são melhores.

**Comment by ClaraKataryna654** [comment]
what level of affection do you think unlocks chess completely

**Comment by satoshi-kun** [comment]
I would say 100 but I'm not sure.
The important thing is that you enjoy your time with her and it will unlock in no time :)
